### PR TITLE
New baselines for CSharpAutoCompleteTest

### PR DESCRIPTION
**Pull request by ajaybhargavb**
https://github.com/aspnet/Razor/issues/2698

### CODE REVIEW

File: src/Microsoft.AspNetCore.Razor.Language/Syntax/SyntaxSerializer.cs [hunk 1]
```diff
@@ -167,6 +172,34 @@ private void WriteNode(SyntaxNode node)
...
 
+            private void WriteRazorDirective(RazorDirectiveSyntax node)
+            {
+                if (node.DirectiveDescriptor == null)
+                {
+                    return;
+                }
+
+                var builder = new StringBuilder("Directive:{");
+                builder.Append(node.DirectiveDescriptor.Directive);
```
Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.Razor.Language/Syntax/SyntaxSerializer.cs]: Decided to write the directive descriptor info to baselines like we did before for `DirectiveChunkGenerator`. Note I'm not using the chunk generator but instead `DirectiveDescriptor` is a first class property on `RazorDirectiveSyntax`.

File: src/Microsoft.AspNetCore.Razor.Language/Syntax/SyntaxSerializer.cs [hunk 2]
```diff
@@ -167,6 +172,34 @@ private void WriteNode(SyntaxNode node)
                 }
             }
 
+            private void WriteRazorDirective(RazorDirectiveSyntax node)
```
Comment (NTaylorMullen) [inline src/Microsoft.AspNetCore.Razor.Language/Syntax/SyntaxSerializer.cs]: q: why can't tests just use this?

Comment (ajaybhargavb) [reply src/Microsoft.AspNetCore.Razor.Language/Syntax/SyntaxSerializer.cs]: Just following what we do for IR nodes. We have two different serializers; one in source used for debugging and one in test used for baselining.

Comment (NTaylorMullen) [reply src/Microsoft.AspNetCore.Razor.Language/Syntax/SyntaxSerializer.cs]: kk

**Comment by ajaybhargavb** [comment]
f32e82fb75e8ee8aa470ba39570c428052ee6f52

**Pull request closed without merging**